Lib/test/test_compileall.py

```diff
@@ -53,8 +53,8 @@ def add_bad_source_file(self):
     def timestamp_metadata(self):
         with open(self.bc_path, 'rb') as file:
             data = file.read(12)
-        mtime = int(os.stat(self.source_path).st_mtime)
-        compare = struct.pack('<4sll', importlib.util.MAGIC_NUMBER, 0, mtime)
+        mtime = int(os.stat(self.source_path).st_mtime) & 0xFFFF_FFF
```

Comment:
That's why I like the underscore: it's easier to spot bugs. You missed something :-)